app/src/main/java/org/mozilla/rocket/content/HomeFragmentViewState.kt

```diff
@@ -0,0 +1,22 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.rocket.content
+
+object HomeFragmentViewState {
+
+    // TODO: make them enum / sealed class
```

Comment:
I really want #3247. If that merged, we don't need to keep the state. Thus no need for this class